services/configuration/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exonum-configuration"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
I'm against releasing configuration until the new functionality (e.g., error handling) is covered with tests. If a release is necessary for some reason, I suggest to make it a release candidate: `0.6.0-rc.1`.